nixos/modules/virtualisation/libvirtd.nix

```diff
@@ -14,6 +14,9 @@ let
     ${cfg.extraConfig}
   '';
   qemuConfigFile = pkgs.writeText "qemu.conf" ''
+    ${optionalString cfg.qemuOvmf ''
+      nvram = ["${pkgs.OVMF}/FV/OVMF_CODE.fd:${pkgs.OVMF}/FV/OVMF_VARS.fd"]
```

Comment:
What about garbage collection? Does it matter if ```${pkgs.OVMF}/FV/OVMF_*``` disappear eventually?

I see that string gets copied to ```/var/lib/libvirt/qemu/DOMAIN.xml```:

```
...
  <os>
    <type arch='x86_64' machine='pc-i440fx-2.8'>hvm</type>
    <loader readonly='yes' type='pflash'>/nix/store/ykpjj629qspk0zrw3f21kv8mw2675asn-OVMF-2014-12-10/FV/OVMF_CODE.fd</loader>
    <nvram>/var/lib/libvirt/qemu/nvram/ubuntu-2_VARS.fd</nvram>
    <boot dev='hd'/>
  </os>
```

For DOMAIN.xml we already have code to refresh emulator path ```<emulator>/nix/store/.../bin/qemu-kvm</emulator>```. Do we have to add this for the ```<loader>``` above too?